src/classes/classes-public.ts

```diff
@@ -0,0 +1,64 @@
+//public classes
+
+/*
+In object-oriented programming (OOP), classes serve as blueprints for creating objects. A class declaration is made using the 'class' keyword, and instances are created using the 'new' keyword followed by the class name.
+
+In TypeScript classes have visibility modifiers:
+
+public (accessible outside the class)
+protected (accessible by instances and subclasses)
+private (inaccessible outside the class).
+*/
+class Person {
+  constructor(public name: string, public age: number) {}
+
+  public getBirthYear() {
+    return new Date().getFullYear() - this.age;
+  }
+}
+const jon = new Person('John', 35);
+
+//class methods
+class Animal {
```

Comment:
I am getting a `Duplicate identifier 'Animal'.` error here.
This happens because of the `Animal` type I created in the file `interfaces-vs-types` (strangely I didn't get the error back then).
Please comment out the lines 30-32 in the before-mentioned file to get rid of the error as the example is still valid if we comment it out.